### ISSUE TITLE
Windows problems

### ISSUE THREAD

**Issue by zdm** [powerman/vim-plugin-viewdoc#54]
Hi, it worked under windows, but now stopped with message "System cannot find the path specified".
Is it possible to fix this, pls?

**Comment by powerman** [comment]
At a glance it's unlikely a viewdoc issue, so yeah, I think so. But I've no Windows, and you didn't provided any details, so I doubt I can help.

**Comment by zdm** [comment]
I have no details, just this message.
Vim messages log is empty, so I can't understand, what file it tries to 
find.

**Comment by powerman** [comment]
Well, viewdoc itself doesn't open any files, AFAIR. It does run external commands - e.g. `go doc …` to show help for Go, `pydoc` and `bash` to show doc for Python, etc.

So, why do you think it's a viewdoc issue at first place, what did you do to trigger this error?
Next question is which one of documentation types you're trying to view (bash/go/python/perl/man/php/etc.) and if other types work?

**Comment by zdm** [comment]
I am trying to view doc for vim module, `:ViewDocHelp crunch`. It works 
under linux, but under windows show new buffer with the error message.

I think, that viewdoc is trying to run some command, but it is not 
present in system or can't be found under windows.

Is it possible to understand whats happens somehow?

**Comment by powerman** [comment]
Is `:help crunch` works?
As for viewdoc code, Vim help is implemented in [plugin/viewdoc_help.vim](https://github.com/powerman/vim-plugin-viewdoc/blob/master/plugin/viewdoc_help.vim). It looks like it executes either `cat` or `zcat`, depending on is target help file is gzip'ed.

**Comment by zdm** [comment]
so, the problem is in this line `execute 'silent 0r ! ( ' . h.cmd . ' ) 
2>/dev/null'`.

need to remove `2>/dev/null` for windows (don;t know, how it can work 
previously).

**Comment by zdm** [comment]
for windows it can be replaced with `2> nul`

**Comment by zdm** [comment]
patch can looks like this:
```
if !has('win16') && !has('win32') && !has('win64')
    execute 'silent 0r ! ( ' . h.cmd . ' ) 2>/dev/null'
else
    execute 'silent 0r ! ( ' . h.cmd . ' ) 2> nul'
endif
```

Could you, please, update the code?

**Comment by powerman** [comment]
I've pushed fix into the master branch. Can you please test it and confirm it fixed the issue?

**Comment by zdm** [comment]
It works now under win and lin.
Thank you.

**Comment by powerman** [comment]
Cool, thanks. Released 1.14.
